docs/pages/routing/installation.mdx

```diff
@@ -29,13 +29,19 @@ Once setup, you can start your project by running:
 
 <Terminal cmd={['$ npx expo start']} />
 
-To view your app on a mobile device, we recommend starting with [Expo Go](/get-started/expo-go). As your application grows in complexity and you need more control, you can create a [development build](/develop/development-builds/introduction/).
+- To view your app on a mobile device, we recommend starting with [Expo Go](/get-started/expo-go). As your application grows in complexity and you need more control, you can create a [development build](/develop/development-builds/introduction/).
+- Open the project in a web browser by pressing <kbd>w</kbd> in the Terminal UI. Press <kbd>a</kbd> for Android (Android Studio is required), or <kbd>i</kbd> for iOS (macOS with Xcode is required).
 
 </Step>
 
 ## Manual installation
 
-> Expo Router supports Expo SDK 48 and above. Some features may require the latest SDK.
+Ensure the version of Expo Router is compatible with the Expo SDK version your project is using.
+
+| Expo SDK | Expo Router |
+| -------- | ----------- |
+| ^49.0.0  | ^2.0.0      |
+| ^48.0.0  | ^1.0.0      |
```

Comment:
```suggestion
| 49.0.0  | 2.0.0 and above     |
| 48.0.0  | 1.0.0      |
```

I think we can remove the caret and specify in words if we want to say, for example, for SDK 49 its `2.0.0` and above.